ompi/mca/mtl/psm2/help-mtl-psm2.txt

```diff
@@ -45,3 +45,7 @@ Unknown path record query mechanism %s. Supported mechanisms are %s.
 #
 [message too big]
 Message size %llu bigger than supported by PSM2 API. Max = %llu
+#
```

Comment:
NOTE: I realize the `message too big` help message is not part of this PR, but I can't help seeing it.  You should not use `%llu` in printf-formatted strings (such as show_help messages).  Instead, you should use the appropriate `PRIfoo` escape sequence.  Please introduce another commit to fix the `message too big` show_help message (per my other comments, you might want to make this message a bit more verbose and clear to the user -- the message as it currently stands does not explain anything about what MPI API call caused this error, etc.).  Thanks!

Reply:
As .txt files are not preprocessed using ```PRIfoo``` would not deliver the expected outcome ?!

Reply:
Er... you're right.  Duh.  Ok, disregard my comment about PRI.  😊